Phonebook.Source.MockPeopleBackend/mocks/synthetic/rooms.json

```diff
@@ -1 +1 @@
e":"East Adah"},{"Building":"8807 Adelia Club","BuildingId":11,"Description":"North Emmittborough, 4221 Jennifer Lock, Raum 5B","Floor":"3","FloorPlan":"sample","Id":"3A","Number":"5B","Phone":"","Place":"North Emmittborough"},{"Building":"429 Kelsi Fork","BuildingId":12,"Description":"Jaidafort, 264 Wyman Tunnel, Raum 1B","Floor":"6","FloorPlan":"sample","Id":"6B","Number":"6A","Phone":"","Place":"Jaidafort"},{"Building":"429 Kelsi Fork","BuildingId":83,"Description":"Sydneyberg, 264 Wyman Tunnel, Raum 6B","Floor":"2","FloorPlan":"sample","Id":"3A","Number":"3A","Phone":"","Place":"Sydneyberg"},{"Building":"5340 Davis Grove","BuildingId":10,"Description":"Lake Tyrell, 429 Kelsi Fork, Raum 2B","Floor":"5","FloorPlan":"sample","Id":"1A","Number":"2A","Phone":"","Place":"Lake Tyrell"},{"Building":"0423 Leila Heights","BuildingId":79,"Description":"Sydneyberg, 449 Katarina Bridge, Raum 6B","Floor":"0","FloorPlan":"sample","Id":"5B","Number":"4A","Phone":"","Place":"Sydneyberg"},{"Building":"742 Marquardt Crossroad","BuildingId":92,"Description":"Ziememouth, 5340 Davis Grove, Raum 1B","Floor":"1","FloorPlan":"sample","Id":"3B","Number":"6A","Phone":"","Place":"Ziememouth"},{"Building":"78132 Toy Park","BuildingId":69,"Description":"Lake Tyrell, 35039 Mante Flats, Raum 4B","Floor":"4","FloorPlan":"sample","Id":"3B","Number":"4A","Phone":"","Place":"Lake Tyrell"},{"Building":"3902 McCullough Oval","BuildingId":81,"Description":"East Quintenville, 06760 Price Throughway, Raum 1B","Floor":"6","FloorPlan":"sample","Id":"2B","Number":"5A","Phone":"","Place":"East Quintenville"},{"Building":"95740 Deanna Cape","BuildingId":40,"Description":"Brodyland, 35039 Mante Flats, Raum 4B","Floor":"4","FloorPlan":"sample","Id":"1A","Number":"2B","Phone":"","Place":"Brodyland"},{"Building":"117 Alverta Rest","BuildingId":100,"Description":"Kleinland, 95740 Deanna Cape, Raum 2A","Floor":"0","FloorPlan":"sample","Id":"4A","Number":"2A","Phone":"","Place":"Kleinland"},{"Building":"264 Wyman Tunnel","BuildingId":91,"Description":"Ziememouth, 9551 Jacobs Stravenue, Raum 5B","Floor":"2","FloorPlan":"sample","Id":"6B","Number":"5B","Phone":"","Place":"Ziememouth"},{"Building":"5340 Davis Grove","BuildingId":46,"Description":"North Emmittborough, 117 Alverta Rest, Raum 6B","Floor":"1","FloorPlan":"sample","Id":"5B","Number":"1B","Phone":"","Place":"North Emmittborough"},{"Building":"3902 McCullough Oval","BuildingId":80,"Description":"East Liaport, 79311 Streich Motorway, Raum 4A","Floor":"6","FloorPlan":"sample","Id":"4A","Number":"3B","Phone":"","Place":"East Liaport"},{"Building":"449 Katarina Bridge","BuildingId":68,"Description":"Aldaview, 8733 Alfonso Port, Raum 3B","Floor":"4","FloorPlan":"sample","Id":"4A","Number":"4A","Phone":"","Place":"Aldaview"},{"Building":"97791 Nikolaus Gateway","BuildingId":83,"Description":"Jaidafort, 97791 Nikolaus Gateway, Raum 4B","Floor":"2","FloorPlan":"sample","Id":"2A","Number":"6B","Phone":"","Place":"Jaidafort"},{"Building":"200 East
```

Comment:
please revert the changes in this file. They are not relevant for the kubernetes deployment